concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    for subtask in Subtask.objects.all():
+        try:
+            report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+            if not isinstance(report_computed_task.size, int):
+                delete_unsupported_messages(subtask)
+        except MessageError as golem_messages_exception:
+            logger.info(f'During message deserialization exception raised: {golem_messages_exception}')
+            delete_unsupported_messages(subtask)
+
+
+def delete_unsupported_messages(subtask: Subtask) -> None:
+    subtask_id = subtask.subtask_id
+    StoredMessage.objects.filter(subtask_id=subtask_id).delete()
+    subtask.delete()
+    logger.info(f'Deleted subtask and all related messages with subtask_id: {subtask_id}')
```

Comment:
Maybe print the whole subtask rather than just the ID?

Reply:
Do we really need this? Should we print whole Subtask with all nested messages?

Reply:
Does `str(subtask)` print all nested messages? If it does then I think that `Subtask.__str__()` prints way too much.